src/Requests/Functions.php

```diff
@@ -51,7 +51,10 @@ function registerNewSchemaVersionWithSubjectRequest(string $schema, string $subj
     return new Request(
         'POST',
         (new UriTemplate())->expand('/subjects/{name}/versions', ['name' => $subjectName]),
-        ['Accept' => 'application/vnd.schemaregistry.v1+json'],
+        [
+            'Accept' => 'application/vnd.schemaregistry.v1+json',
+            'Content-Type' => 'application/vnd.schemaregistry.v1+json'
+        ],
```

Comment:
Nitpick: Could we refactor this into a constant `MEDIA_TYPE_HEADERS` and replace the duplicated array?

Reply:
i added a commit to change this, something like this?

Reply:
See the other comment regarding the structure of the constants.